Frameworks/CoreGraphics/CGFont.mm

```diff
@@ -40,6 +41,9 @@
     CFRuntimeBase _base;
     ComPtr<IDWriteFontFace> _dwriteFontFace;
 
+    // Contains a value when created via Data Provider, null o/w
+    woc::unique_cf<CFDataRef> _data;
```

Comment:
hmm, wondering if it's worth it to cache the data provider instead here. our current data provider impl is just a data, but we could save some memory in the future once data provider is properly implemented and defers loading its binary data